src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -587,6 +593,18 @@ static PyMethodDef Message_methods[] = {
 	  "  :rtype: (int, int)\n"
 	  "\n"
 	},
+	{ "latency", (PyCFunction)Message_latency, METH_NOARGS,
+          "Retrieve the time it took to produce the message, from calling "
+          "produce() to the time the acknowledgement was received from "
+          "the broker.\n"
+          "Must only be used with the producer for successfully produced "
```

Comment:
> Must only be used with the producer for successfully produced messages.

doesn't really make sense i don't think, and i don't think you need to state this. should just return None if not applicable.

Reply:
Why doesnt it make sense to state this her?
Otherwise people will try to use it out of scope (consumer side, or for failed messages)  and wonder why it doesnt work